src/task.rs

```diff
@@ -1220,6 +1233,13 @@ pub struct TaskMetrics {
     /// ```
     pub total_slow_poll_count: u64,
 
+    /// The total count of tasks with long scheduling delays.
+    ///
+    /// This is defined as tasks taking longer than
+    /// [`long_delay_threshold`][TaskMonitor::long_delay_threshold] to be executed aftering being
+    /// scheduled.
```

Comment:
Can you follow the documentation pattern of [`total_slow_poll_count`](https://docs.rs/tokio-metrics/0.2.0/tokio_metrics/struct.TaskMetrics.html#structfield.total_slow_poll_count) here?